.travis.yml

```diff
@@ -20,9 +20,9 @@ before_script:
     - composer install
 
 script:
-    - ./vendor/bin/phpunit --coverage-clover=coverage.clover
-    - sh -c 'if [ "$TRAVIS_PHP_VERSION" != "hhvm-3.30" ]; then wget https://scrutinizer-ci.com/ocular.phar; fi;'
-    - sh -c 'if [ "$TRAVIS_PHP_VERSION" != "hhvm-3.30" ]; then php ocular.phar code-coverage:upload --format=php-clover coverage.clover; fi'
-    - sh -c 'if [ -n "$CODECLIMATE_REPO_TOKEN" -a "$TRAVIS_PHP_VERSION" != "hhvm-3.30" ]; then vendor/bin/test-reporter --coverage-report=coverage.clover; fi'
+ - ./vendor/bin/phpunit --coverage-clover=coverage.clover
 
-sudo: false
```

Comment:
Why remove the sudo line here? My understanding is that travis can run in a more optimized way if it doesn't require sudo.

Reply:
The `sudo: false` is disabled by default in Travis CI.

It's fine to remove this setting.

The reference link is available [here](https://docs.travis-ci.com/user/reference/trusty/#container-based-infrastructure).

Reply:
Excellent. Thanks Peter. Top-notch work!